commands/modmail.js

```diff
@@ -0,0 +1,49 @@
+import { SlashCommandBuilder } from "@discordjs/builders";
+import dotenv from "dotenv";
+
+dotenv.config();
+const { MODMAIL_CHANNEL } = process.env;
+if (!MODMAIL_CHANNEL) throw new Error("MODMAIL_CHANNEL is not set in the .env.");
+
+/** @type {import("../types/command").default} */
+const info = {
+	data: new SlashCommandBuilder()
+		.addSubcommand((subcommand) =>
+			subcommand
+				.setName("close")
+				.setDescription("(Mods Only) Send a message in a modmail thread"),
+		)
+		.setDescription("ModMail Commands"),
+
+	async interaction(interaction) {
+		if (
+			interaction.channel?.type !== "GUILD_PUBLIC_THREAD" ||
+			interaction.channel.parentId !== MODMAIL_CHANNEL
+		)
+			return;
+		const command = interaction.options.getSubcommand();
+		if (command === "close") {
+			await interaction.reply({
+				content: `:white_check_mark: ModMail closed!`,
+			});
+			const starter = await interaction.channel.fetchStarterMessage();
+			const user = await interaction.client.users.fetch(starter.embeds[0]?.description || "");
+			if (!user) return;
+			const dm = await user.createDM();
+			dm.send("ModMail closed!");
+
+			await starter.edit({
+				embeds: [
+					{
+						title: "ModMail Ticket Closed",
+						description: starter.embeds[0]?.description || "",
+						color: 0x00ff00,
+					},
+				],
+			});
+			interaction.channel.delete("Closed by " + interaction.user.username);
```

Comment:
dont delete just archive and lock

Reply:
@RedGuy12 archive and lock still allows mods to surpass it

Reply:
mods dont have manage threads, only admins do

I see your point tho

the thing is, it'd be good to have a record of modmails